### PR TITLE
Add instructions re: SXGs using Web Packager

### PR DESCRIPTION
**Pull request by khempenius**

### CODE REVIEW

File: src/site/content/en/fast/signed-exchanges-webpackager/index.md [hunk 1]
```diff
@@ -0,0 +1,465 @@
...
+description: |
+  Learn how to serve signed exchanges (SXGs) using Web Packager.
+tags:
+  - blog
+  - performance
+---
+
+The following instructions explain how to set up Signed Exchanges (SXGs) using
+Web Packager. Instructions are included for both self-signed and
+`CanSignHttpExchanges` certificates.
```
Comment (mihajlija) [inline src/site/content/en/fast/signed-exchanges-webpackager/index.md:20]:
```suggestion
The following instructions explain how to set up [Signed Exchanges (SXGs)](/signed-exchanges) using
[Web Packager](https://github.com/google/webpackager). Instructions are included for both self-signed and
`CanSignHttpExchanges` certificates.
```

File: src/site/content/en/fast/signed-exchanges-webpackager/index.md [hunk 2]
```diff
@@ -0,0 +1,465 @@
...
+    site.
+
+3. Create a certificate that has the `CanSignHttpExchanges` extension.
+
+    ```shell
+    openssl x509 -req -days 90 -in cert.csr -signkey priv.key -out cert.pem -extfile &lt;(echo -e "1.3.6.1.4.1.11129.2.1.22 = ASN1:NULL\nsubjectAltName=DNS:example.com")
+    ```
+
+    This command uses the private key and CSR created in steps 1 and 2 to create the
+    certificate file `cert.pem`. The `-extfile` flag associates the certificate with
```
Comment (mihajlija) [inline src/site/content/en/fast/signed-exchanges-webpackager/index.md:101]:
```suggestion
    This command uses the private key and the CSR created in steps 1 and 2 to create the
    certificate file `cert.pem`. The `-extfile` flag associates the certificate with
```

File: src/site/content/en/fast/signed-exchanges-webpackager/index.md [hunk 3]
```diff
@@ -0,0 +1,465 @@
...
+        article verbatim, this should be changed to `example.com`. `webpkgserver`
+        will only fetch content from the domain indicated by `webpkgserver.toml`. If
+        you try to fetch pages from a different domain without updating
+        `webpkgserver.toml`, the `webpkgserver` logs will show the error message
+        `URL doesn't match the fetch targets`.
+    *   (Optional) To preload subresources, change the line `#PreloadCSS
+        = false` to `PreloadCSS = true`. In addition, change the line `#PreloadJS =
+        false` to `PreloadJS = true`. More information about subresource
+        substitution can be found
+        [here](https://github.com/WICG/webpackage/blob/master/explainers/signed-exchange-subresource-substitution.md).
```
Comment (mihajlija) [inline src/site/content/en/fast/signed-exchanges-webpackager/index.md:201]:
```suggestion
    *   (Optional) To preload subresources, change the line `#PreloadCSS
        = false` to `PreloadCSS = true`. In addition, change the line `#PreloadJS =
        false` to `PreloadJS = true`. For more information about subresource
        substitution, check out
        [the explainer](https://github.com/WICG/webpackage/blob/master/explainers/signed-exchange-subresource-substitution.md).
```

File: src/site/content/en/fast/signed-exchanges-webpackager/index.md [hunk 4]
```diff
@@ -0,0 +1,465 @@
...
+  request that the content be delivered without using SXG.
+
+  These instructions explain how to serve a certificate from an existing HTTPS   environment. 
+
+  {% Aside %}
+  `webpkgserver` can be configured to use a locally-hosted
+  certificate. For information on this configuration option,
+  refer to the `CertURLBase` option in `webpkgserver.toml`. This 
+  [article](https://web.dev/how-to-use-local-https/) explains how to 
+  use HTTPS for local development.
```
Comment (mihajlija) [inline src/site/content/en/fast/signed-exchanges-webpackager/index.md:289]:
```suggestion
  `webpkgserver` can be configured to use a locally-hosted
  certificate. For information on this configuration option,
  refer to the `CertURLBase` option in `webpkgserver.toml`. 
  Learn more about [how to use HTTPS for local development](https://web.dev/how-to-use-local-https/).
```

File: src/site/content/en/fast/signed-exchanges-webpackager/index.md [hunk 5]
```diff
@@ -0,0 +1,465 @@
...
+  regenerate the SXG using a different certificate, the **Certificate URL**
+  listed in the **Signature** would be different.
+
+  {% Img
+    src="image/j2RDdG43oidUy6AL6LovThjeX9c2/K1caujHPlyCURA1pxYXz.png",
+    alt="Screenshot showing the hash that corresponds to the certificate.",     width="800",
+    height="229"
+  %}
+
+2. Create a copy of `cert.pem`. The filename of the new version should match the
```
Comment (mihajlija) [inline src/site/content/en/fast/signed-exchanges-webpackager/index.md:317]: Seems like the image here breaks the continuity of the numbered list, the steps are rendered as 1, 1, 2, etc

**Review by mihajlija — APPROVED** [review]

**Comment by mihajlija** [comment]
LGTM + I think you can/should now upload the hero image to GCP too, see #4703

File: src/site/content/en/fast/signed-exchanges-webpackager/index.md [hunk 6]
```diff
@@ -0,0 +1,467 @@
...
+{% Img 
+  src="image/j2RDdG43oidUy6AL6LovThjeX9c2/cyV0pNmlTB4PTCc5YQe3.png",
+  alt="Signed Exchanges are loaded from a Web Packager instance running 
+  on localhost:8080",
+  width="800",
+  height="436"
+%}
+
+
+Following these instructions verbatim will setup a `webpackager` instance that
```
Comment (banaag) [inline src/site/content/en/fast/signed-exchanges-webpackager/index.md:53]: `webpackager` HTTP server instance

File: src/site/content/en/fast/signed-exchanges-webpackager/index.md [hunk 7]
```diff
@@ -0,0 +1,467 @@
...
+
+    ```shell
+    cp ./webpkgserver.example.toml ./webpkgserver.toml
+    ```
+
+    This file contains the configuration options for `webpkgserver`.
+
+
+
+3. Open `webpkgserver.toml` with an editor of your choice and make the following
```
Comment (banaag) [inline src/site/content/en/fast/signed-exchanges-webpackager/index.md:183]: You might want to mention this optional TOML config:  KeepNonSXGPreloads (default = false).
  # Include preload link headers that lack the corresponding allowed-alt-sxg
  # with a valid header-integrity.

The TOML describes what it's for.

File: src/site/content/en/fast/signed-exchanges-webpackager/index.md [hunk 8]
```diff
@@ -0,0 +1,467 @@
...
+
+    If this header is not set, you will see the following error when you inspect the
+    SXG in DevTools: `Content type of cert-url must be application/cert-chain+cbor.
+    Actual content type: text/html`.
+
+4. Restart `webpkgserver`.
+
+    ```shell
+    webpkgserver
+    ```
```
Comment (banaag) [inline src/site/content/en/fast/signed-exchanges-webpackager/index.md:467]: As an optional step, you can also talk about how webpkgserver can auto-renew the SXG certs every 90 days if they have a DigiCert ACME account setup.  Relevant config items can be found here: https://github.com/google/webpackager/blob/master/cmd/webpkgserver/webpkgserver.example.toml#L228

File: src/site/content/en/fast/signed-exchanges-webpackager/index.md [hunk 9]
```diff
@@ -0,0 +1,467 @@
...
+
+    Certificates come in many formats. `cert.pem` is in the [PEM
+    format](https://en.wikipedia.org/wiki/Privacy-Enhanced_Mail). Certificates for
+    signed exchanges must be in the `application/cert-chain+cbor` format.
+
+    When using a self-signed certificate, `webpkgserver` automatically converts
+    the certificate indicated by the `PEMFile` option in `webpkgserver.toml` to
+    the `application/cert-chain+cbor` format. However, if you are using a
+    `CanSignHttpExchanges` certificate, you must generate the CBOR-encoded
+    certificate yourself.
```
Comment (twifkak) [inline src/site/content/en/fast/signed-exchanges-webpackager/index.md:434]: I think this section is incorrect and should be removed. Web Packager Server will generate cert-chain+cbor even for CanSignHttpExchanges certificates.

(There is a [Web Packager CLI](https://github.com/google/webpackager) that requires use of gen-certurl, but that's not talked about in this doc, so I think safe to omit.)

File: src/site/content/en/fast/signed-exchanges-webpackager/index.md [hunk 10]
```diff
@@ -0,0 +1,467 @@
...
+### Setup up Web Packager for production use
+
+
+
+1. Follow steps 1 and 2 of [Setup the Web Packager Server for
+   testing](/signed-exchanges-webpackager/#setup-the-web-packager-server-for-testing).
+
+2. Open `webpkgserver.toml` with the editor of your choice and make the
+   following changes:
+    *   Change the line `PEMFile = cert.pem` to `PEMFile = cert.cbor`.
```
Comment (twifkak) [inline src/site/content/en/fast/signed-exchanges-webpackager/index.md:446]: This line should be changed. PEMFile should point to a PEM file that contains the concatenation of your site's SXG certificate and your CA's certificate.

File: src/site/content/en/fast/signed-exchanges-webpackager/index.md (hunk 8 above)
Comment (twifkak) [inline src/site/content/en/fast/signed-exchanges-webpackager/index.md:467]: Likewise, this section can be omitted as it only applies to the Web Packager CLI, not the server.

**Review by twifkak — COMMENTED** [review]
Hi! Hope it's OK to leave a few comments here. I think there are some necessary updates.

File: src/site/content/en/fast/signed-exchanges-webpackager/index.md (hunk 7 above)
Comment (twifkak) [reply src/site/content/en/fast/signed-exchanges-webpackager/index.md:183]: If you mention this option, you might want to disclaim that doing so could make the SXG ineligible for the Google SXG cache per [these requirements](https://github.com/google/webpackager/blob/master/docs/cache_requirements.md).

**Pull request merged**